modules/core/src/main/lilaism/LilaLibraryExtensions.scala

```diff
@@ -106,6 +106,24 @@ trait LilaLibraryExtensions extends CoreExports:
           acc.flatMap: _ =>
             f(a).void
 
+    /** traverse the list sequentially, short-circuiting on the first error.
+      *
+      * returning the first error if there is and the successfully processed elements
+      */
+    def sequentiallyRaise[E, B](f: A => FuRaise[E, B])(using EC): Fu[(List[B], Option[E])] =
```

Comment:
just to be sure, this doesn't reverse the list?

Reply:
my mistake 🥹